tox.ini

```diff
@@ -4,21 +4,19 @@
 minversion = 3.18
 envlist =
     lint
-    py27
-    py35
-    py36
     py37
     py38
     py39
     py310
     py311
-    pypy
     pypy3
     docs
     coverage
 
 [testenv]
 usedevelop = true
+package = wheel
+wheel_build_env = .pkg
```

Comment:
Out of curiosity, what's that for?

Reply:
It is taken from https://hynek.me/articles/turbo-charge-tox/ to speed up build time.
On my local machine before adding it a tox run of a single Python version took 4.3 s now it takes 1.1 s.

Reply:
See also https://github.com/zopefoundation/meta/pull/201

Reply:
😱  Are you adding that to meta/config?